packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -289,6 +292,17 @@ function restLevelPackage(model: RLCModel, hasSamplesGenerated: boolean) {
       packageInfo.scripts["integration-test:node"] =
         "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'";
     }
+    if (isTypeSpecTest) {
+      packageInfo["mocha"] = {
+        extension: ["ts"],
+        timeout: "1200000",
+        loader: "ts-node/esm"
+      };
+      packageInfo.scripts["unit-test:node"] =
+        'mocha --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace "test/{,!(browser)/**/}*.spec.ts"';
+      packageInfo.scripts["integration-test:node"] =
+        'nyc mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace "dist-esm/test/{,!(browser)/**/}*.spec.js"';
```

Comment:
I feel like this common/tools/mocha-multi-reporter.js is from the js repo, are you sure this only applies to isTypeSpecTest not azureSdkForJs ？

Reply:
Good point! azureSDKForJs is using `dev-tool run` command which has these baked in already.  Would removing `--reporter ../../../common/tools/mocha-multi-reporter.js` good enough? I think it's for having a couple test result report in JS repo

Reply:
Mocha will still output the result to the console.  With common/tools/mocha-multi-reporter.js in js-repo, mocha also outputs test results to some files, our build pipelines can then upload them to their associated build so we can see the results in the Tests tab.